lib/seed/Seeder.js

```diff
@@ -41,22 +44,25 @@ class Seeder {
   async _listAll(config) {
     this.config = this.setConfig(config);
     const loadExtensions = this.config.loadExtensions;
-    return readdir(this._absoluteConfigDir()).then((seeds) =>
-      seeds
-        .filter((value) => {
-          const extension = path.extname(value);
-          return includes(loadExtensions, extension);
-        })
-        .sort()
-    );
+    const seeds = (
+      await Promise.all(
+        this._absoluteConfigDirs().map(getFilepathsInFolderRecursively)
+      )
+    ).flat();
+    return seeds
+      .filter((value) => {
```

Comment:
I would assume this filter function is used in other places as well, probably we can extract it.